### PR TITLE
Reformat JSON, Add new resources, Remove Autoprefixer from css.json

### PR DESCRIPTION
**Pull request by tomhodgins**
The reason I say to remove autoprefixer is browsers dont use vendor prefixes anymore except for `-webkit-*` so that makes a tool that automatically adds legacy prefixes to stuff more like "auto-mess-up-my-code-please".

Reformatted JSON to be consistent in indentation/spacing.

Added a bunch of new things, especially to CSS <3

### CODE REVIEW

File: categories/git.json [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+    "description": "Game-like website to learn and master git.",
+    "url": "https://learngitbranching.js.org",
+    "free": true,
+    "tags": [
+      "git",
+      "game"
+    ]
+  },
+  {
+    "name": "Oh Shit, Git!?!",
```
Comment (LamboCreeper) [inline categories/git.json]: Can we censor this to "S***" please (family friendly website)!

File: categories/javascript.json [hunk 2]
```diff
@@ -31,6 +31,28 @@
...
+    "url": "https://gomakethings.com/ditching-jquery/",
+    "free": true,
+    "tags": [
+      "jquery",
+      "javascript",
+      "vanilla"
+    ]
+  },
+  {
+    "name": "HtmlDom.dev",
```
Comment (LamboCreeper) [inline categories/javascript.json]: From a formatting standpoint should HTML and DOM not be capitalised?

**Review by LamboCreeper — COMMENTED** [review]
Really appreciate all of these additions!

File: categories/git.json (hunk 1 above)
Comment (tomhodgins) [reply categories/git.json]: Done, and updated the link too - hope it handles the encoded URL correctly 😇

File: categories/javascript.json (hunk 2 above)
Comment (tomhodgins) [reply categories/javascript.json]: Updated this to `htmldom.dev` which is the same styling used on the github repository for that project

File: categories/git.json (hunk 1 above)
Comment (LamboCreeper) [reply categories/git.json]: That’s super cool!

**Review by LamboCreeper — APPROVED** [review]

File: categories/css.json [hunk 3]
```diff
@@ -70,5 +63,147 @@
...
+    "description": "Browser support chart showing current support for CSS technologies under development.",
+    "url": "https://css-houdini.rocks/",
+    "free": true,
+    "tags": [
+      "css",
+      "houdini",
+      "support",
+      "reference"
+    ]
+  },
```
Comment (ajsaraujo) [inline categories/css.json]: Wrong URL for the second resource! I guess you just forgot to copy the second one and pasted the same URL twice haha

**Review by ajsaraujo — APPROVED** [review]

**Pull request merged**